Cargo.toml

```diff
@@ -52,7 +52,7 @@ path = "examples/deferred/main.rs"
 name = "flowmap"
 path = "examples/flowmap/main.rs"
 
-[[target.x86_64-pc-linux-gnu.example]]
+[[example]]
```

Comment:
AFAIK, having GLFW dependency on Windows is an eternal pain...

> On Apr 25, 2016, at 2:51, Eugene Gorodinsky <notifications@github.com> wrote:
> 
> In Cargo.toml:
> 
> > @@ -52,7 +52,7 @@ path = "examples/deferred/main.rs"
> >  name = "flowmap"
> >  path = "examples/flowmap/main.rs"
> >  
> > -[[target.x86_64-pc-linux-gnu.example]]
> > +[[example]]
> Could we go the opposite way and make it build on windows? I'm using linux, so I can't really test anything, though.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>